_posts/2015-09-21-six-lines-to-install-and-start-sparkr-on-mac-os-x-yosemite.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Six lines to install and start SparkR on Mac OS X Yosemite
+excerpt: for both  and command line
+categories: articles
+tags: [R, Spark, SparkR, Hadoop, big data]
+comments: true
+share: true
+published: true
```

Comment:
can you change it to false, so I can preview it on locally before we publish it...